### PR TITLE
Config panel for Config Entries

### PR DESCRIPTION
**Pull request by balloob**
Video of this in action https://youtu.be/_qpKe-FW9Yk

Config panel for new config entry support.

image

Requires backend PR: https://github.com/home-assistant/home-assistant/pull/12079

Add to your config:

```yaml
config:
  config_entries: true
```

More info here: https://github.com/home-assistant/home-assistant/issues/11947

The UI is very ugly. That is… by design 😉 For now all functionality is there, will polish it further later.

### CODE REVIEW

**Comment by balloob** [comment]
Bump

File: panels/config/config-entries/ha-config-flow.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    if (!step.errors) step.errors = {};
+    this.step = step;
+    // We got a new form if there are no errors.
+    if (Object.keys(step.errors).length === 0) {
+      this.stepData = {};
+    }
+  }
+
+  _flowDone() {
+    this.fire('flow-closed', {
```
Comment (c727) [inline panels/config/config-entries/ha-config-flow.html]: `<link rel='import' href='../../../src/util/hass-mixins.html'>`

File: panels/config/config-entries/ha-config-entries.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+{
+  class HaConfigManager extends window.hassMixins.NavigateMixin(Polymer.Element) {
+    static get is() { return 'ha-config-entries'; }
+
+    static get properties() {
+      return {
+        hass: Object,
+        isWide: Boolean,
+
+        route: Object,
```
Comment (c727) [inline panels/config/config-entries/ha-config-entries.html]: not used

File: panels/config/config-entries/ha-config-entries.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+<script>
+{
+  class HaConfigManager extends window.hassMixins.NavigateMixin(Polymer.Element) {
+    static get is() { return 'ha-config-entries'; }
+
+    static get properties() {
+      return {
+        hass: Object,
+        isWide: Boolean,
```
Comment (c727) [inline panels/config/config-entries/ha-config-entries.html]: can be removed from all files

File: panels/config/config-entries/ha-config-entries.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        <div class='entries layout horizontal wrap'>
+          <template is='dom-repeat' items='[[_entries]]'>
+            <paper-card heading='[[item.title]]'>
+              <div class='card-content'>
+                Integration: [[item.domain]]<br>
+                State: [[item.state]]<br>
+                Added by: [[item.source]]
+              </div>
+              <div class="card-actions">
+                <paper-button on-tap='_removeEntry'>REMOVE</paper-button>
```
Comment (c727) [inline panels/config/config-entries/ha-config-entries.html]: `Remove`

File: panels/config/config-entries/ha-config-entries.html (hunk 3 above)
Comment (balloob) [reply panels/config/config-entries/ha-config-entries.html]: It's used by `ha-config-section` but is indeed not used here, removed it 👍

**Comment by andrey-git** [comment]
This is broken for me. When I open `/config/dashboard` I get
```
GET http://localhost:8123/api/config/config_entries/entry 404 (Not Found)
```
I am at head in both repos, tried both dev and prod mode.

I have 
```
config:
```
in my `configuration.yaml`

Should `components/config/config_entries.py` be auto-loaded?

**Comment by balloob** [comment]
Ugh. This is an issue with how `<iron-pages>` work in `ha-panel-config.html`. It is instantiating all elements that can be seen in the config panel. `ha-push-notification-toggle.html` is also causing an issue if not setup.

Think we should migrate to the dynamicContentUpdater for config?

**Pull request merged**